File: Models/Pizza.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TaviscaAssessment.Models
{
    public class Pizza : Entity
    {
        public int TotalCost { get; set; }

        public int PizaaTypeId { get; set; }

        public string Ingredients { get; set; }

        public ICollection<Order> Order { get; set; }
    }
}

File: Frontend/Pizzeria/src/app/Shared/model/pizza.ts
export class Pizza {
    id: number;
    totalCost: number;
    pizzaTypeId: number;
    Ingredients: string;

    constructor(pizza) {
        {
            this.id = pizza.id;
            this.totalCost = pizza.totalCost || '';
            this.pizzaTypeId = pizza.pizzaTypeId;
            this.Ingredients = pizza.Ingredients || '';
        }
    }
}